src/Tile/Tile.Component.js

```diff
@@ -74,24 +74,50 @@ export const TileComponent = () => {
                 </Tile>
             </Example>
 
+            <Example
+                title='Media with Actions Tile'>
+                <Tile>
+                    <Tile.Media>
+                        <Image photo='https://placeimg.com/400/400/nature' size='l'
+                            type='circle' />
+                    </Tile.Media>
+                    <Tile.Content title='Tile Title' />
+                    <Tile.Actions>
+                        <Popover
+                            body={
+                                <Menu>
+                                    <Menu.List>
+                                        <Menu.Item url='/'>Option 1</Menu.Item>
+                                        <Menu.Item url='/'>Option 2</Menu.Item>
+                                        <Menu.Item url='/'>Option 3</Menu.Item>
+                                        <Menu.Item url='/'>Option 4</Menu.Item>
+                                    </Menu.List>
+                                </Menu>
+                            }
+                            control={<Button glyph='vertical-grip' option='light' />}
+                            placement='bottom-end' />
+                    </Tile.Actions>
+                </Tile>
+            </Example>
+
             <Example
                 title='Product Tile'>
                 <div>
-                    <ProductTile role='button'>
-                        <ProductTile.Media image='https://techne.yaas.io/images/product-thumbnail-wide.png' />
-                        <ProductTile.Content title='Tile Title'>
+                    <Tile productTile role='button'>
+                        <Tile.Media backgroundImage='https://placeimg.com/820/480/any' />
```

Comment:
I think we might need to bring in an image that's consistent for this - or at least figure out the best dimensions here and maybe limit to nature? It's get's a little funky looking on wide screens. 

![Screen Shot 2019-06-03 at 8 40 41 AM](https://user-images.githubusercontent.com/29607818/58814843-53192680-85db-11e9-9036-d1907beb9ebb.png)

Reply:
Sounds good - Looks like 4:1 is a good ratio for header images and will switch it to nature too 👍